test/JDBC/expected/BABEL-3549.out

```diff
@@ -38,7 +38,7 @@ CREATE USER babel_3549_login1
 GO
```

Comment:
Please add follow testcase(for normal run and upgrade):
```
-- run as sysadmin login:
create login smith with password= '123'
go
create database mydb
go
alter server role sysadmin add member smith
go


-- run as login smith
use mydb
go 
select db_name(), suser_name(), user_name()
go


-- run as sysadmin:
use mydb
go
create user smith
go

-- reconnect with login smith:
use mydb
go
select db_name(), suser_name(), user_name()
go

```

Reply:
Added tests.